### PR TITLE
add last release date of rg to download page

### LINKED ISSUE #978: https://rubygems.org/pages/download page should have date of last release of rg
Summary says it all.  I think people at a glance like to know how when the last release of something was put out.

### PR DESCRIPTION
**Pull request by farukaydin**
Fixes #978

cc // @qrush @sferik

### CODE REVIEW

**Comment by dwradcliffe** [comment]
:+1:

File: app/models/rubygem.rb [hunk 1]
```diff
@@ -95,10 +95,9 @@ def self.by_downloads
   end
 
   def self.current_rubygems_release
-    if (g = find_by(name: "rubygems-update")) && (v = g.versions.release.indexed.latest.first)
-      v.number
-    else
-      "0.0.0"
+    if (g = find_by(name: "rubygems-update")) &&
```
Comment (qrush) [inline app/models/rubygem.rb]: `g` is an awful variable name. also shouldn't need the parens around either of these statements. How about `rubygem`?

File: app/views/pages/download.html.erb [hunk 2]
```diff
@@ -1,15 +1,17 @@
 <% @title = "Download RubyGems" %>
-<% ver = Rubygem.current_rubygems_release %>
-<% @subtitle = "v#{ver}" %>
+<% version = Rubygem.current_rubygems_release %>
+<% version_number = version.try(:number) || "0.0.0" %>
+<% @subtitle = "v#{version_number}" %>
+<% @subtitle += " - #{nice_date_for(version.built_at)}" if version.try(:built_at) %>
```
Comment (qrush) [inline app/views/pages/download.html.erb]: These should all be in a helper. Way too much logic for a view.

File: app/views/pages/download.html.erb [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
 <div class="l-colspan t-body">
   <p>
     RubyGems is a package management framework for Ruby. Download the latest version here:
   </p>
   <div id="formats" class="download__formats">
-    <%= link_to "tgz", "http://production.cf.rubygems.org/rubygems/rubygems-#{ver}.tgz", :class => "download__format" %>
-    <%= link_to "zip", "http://production.cf.rubygems.org/rubygems/rubygems-#{ver}.zip", :class => "download__format" %>
-    <%= link_to "gem", "http://production.cf.rubygems.org/rubygems/rubygems-update-#{ver}.gem", :class => "download__format" %>
+    <%= link_to "tgz", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.tgz", :class => "download__format" %>
```
Comment (qrush) [inline app/views/pages/download.html.erb]: Also can use 1.9 hash syntax here. Hashrockets :-1:

**Comment by qrush** [comment]
Had a few comments. Let's keep the codebase clean :grin:

File: app/models/rubygem.rb [hunk 4]
```diff
@@ -95,10 +95,9 @@ def self.by_downloads
   end
 
   def self.current_rubygems_release
-    if (g = find_by(name: "rubygems-update")) && (v = g.versions.release.indexed.latest.first)
-      v.number
-    else
-      "0.0.0"
+    if (rubygem = find_by(name: "rubygems-update")) &&
```
Comment (farukaydin) [inline app/models/rubygem.rb]: @qrush I checked that you said, but first parenthesis are needed. So I deleted second one.

**Comment by farukaydin** [comment]
@qrush Thanks for review :+1:  I fixed and updated like you mentioned.

**Comment by qrush** [comment]
:ok_hand:

**Pull request merged**